### PR TITLE
Make material renderer available for arbitrary item

### PR DESCRIPTION
**Pull request by miozune**
Apply renderer to combs

### CODE REVIEW

**Comment by miozune** [comment]
I'm not sure if we should keep the name `GT_GeneratedItem_Renderer` now that it accepts any kind of item... But like `GT_CustomItemRenderer` does not make difference with other renderers like `GT_FlaskRenderer`

File: src/main/java/gregtech/common/render/items/UniversiumRenderer.java [hunk 1]
```diff
@@ -294,11 +297,6 @@ public IIcon getStackIcon(ItemStack stack, EntityPlayer player) {
     }
 
     public IIcon getTrueIcon(ItemStack stack) {
```
Comment (OneEyeMaker) [inline src/main/java/gregtech/common/render/items/UniversiumRenderer.java:299]: This method should include render pass as argument.

**Review by OneEyeMaker — CHANGES_REQUESTED** [review]

**Review by OneEyeMaker — APPROVED** [review]

File: src/main/java/gregtech/common/render/items/UniversiumRenderer.java [hunk 2]
```diff
@@ -50,47 +50,50 @@ public boolean renderFluidDisplayItem(ItemRenderType type, ItemStack aStack, Obj
...
             }
-        }
 
-        GL11.glDisable(GL11.GL_BLEND);
+            GL11.glDisable(GL11.GL_BLEND);
 
-        if (tIcon != null) {
-            magicRenderMethod(type, aStack, getTrueIcon(aStack), data);
+            if (tIcon != null) {
+                magicRenderMethod(type, aStack, getTrueIcon(aStack), data);
```
Comment (OneEyeMaker) [inline src/main/java/gregtech/common/render/items/UniversiumRenderer.java:90]: Render pass also should be included in this call, shouldn't it?

Comment (miozune) [reply src/main/java/gregtech/common/render/items/UniversiumRenderer.java:90]: Oh you're right. I'm actually working on another PR, will include the fix in it

**Pull request merged**